<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Медицинская система управления процедурами</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>Загрузить CSV</string>
        </property>
       </widget>
      </item>
       <item>
    <widget class="QPushButton" name="createDemoButton">
      <property name="text">
        <string>Создать демо CSV</string>
      </property>
    </widget>
    </item>
      <item>
       <widget class="QComboBox" name="patientsCombo"/>
      </item>
      <item>
       <widget class="QPushButton" name="detailsButton">
        <property name="text">
         <string>Показать детали</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="detailsLabel">
      <property name="text">
       <string>Детали будут здесь...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ProgressWidget" name="progressWidget"/>
    </item>
    <item>
     <widget class="QTableWidget" name="dataTable">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Фамилия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Диагноз</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Процедура</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Параметры</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgressWidget</class>
   <extends>QWidget</extends>
   <header>progresswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>